package.json

```diff
@@ -3,7 +3,7 @@
   "private": true,
   "type": "module",
   "engines": {
-    "node": "20.x",
+    "node": "22.x",
```

Comment:
you can set `>=20` if you can't get your machine to work on 20. strict 22 wont work on vercel